Cargo.toml

```diff
@@ -33,6 +33,7 @@ ruffle_core = { git = "https://github.com/ruffle-rs/ruffle.git", branch = "maste
     "lzma",
     "default_compatibility_rules",
     "default_font",
+    "jpegxr"
```

Comment:
We should enable it in the build script, or this forces every developer to figure out how to set up jpegxr too

Reply:
Ah good point! Should be fixed now